packages/manager/apps/web/client/app/private-database/order/private-sql/private-database-order-private-sql.controller.js

```diff
@@ -0,0 +1,43 @@
+import get from 'lodash/get';
+
+export default class PrivateDatabaseOrderPrivateSqlCtrl {
+  /* @ngInject */
+  constructor($timeout, $translate, Alerter) {
+    this.$timeout = $timeout;
+    this.$translate = $translate;
+    this.Alerter = Alerter;
+  }
+
+  $onInit() {
+    this.alerts = {
+      main: 'app.private-database.order',
+    };
+  }
+
+  onError(error) {
+    this.Alerter.error(
+      this.$translate.instant('private_database_order_private_sql_error', {
+        message: get(error, 'data.message', error),
+      }),
+      this.alerts.main,
+    );
+    this.scrollToTop();
+  }
+
+  onSuccess(url) {
+    this.Alerter.success(
+      this.$translate.instant('private_database_order_private_sql_success', {
+        orderUrl: url,
+      }),
+      this.alerts.main,
+    );
+    this.scrollToTop();
+  }
+
+  scrollToTop() {
+    this.$timeout(() => {
```

Comment:
We can use native builtin Angular to do that.
I do similare stuff here:
packages/manager/modules/pci/src/projects/project/instances/instance/reinstall/reinstall.controller.js (goToEdintAnInstance function)